File: lib/models/auth_response_model.dart
class AuthResponseModel {
  int statusCode;
  String message;
  String accessToken;
  int loginTime;
  int expirationDuration;

  AuthResponseModel({
    required this.statusCode,
    required this.message,
    required this.accessToken,
    required this.loginTime,
    required this.expirationDuration,
  });

  factory AuthResponseModel.fromJson(Map<String, dynamic> json) =>
      AuthResponseModel(
        statusCode: json['statusCode'],
        message: json['message'],
        accessToken: json['accessToken'],
        loginTime: json['loginTime'],
        expirationDuration: json['expirationDuration'],
      );
}

File: backend/ReservationBus/src/main/java/com/busBooking/ReservationBus/models/AuthResponseModel.java
package com.busBooking.ReservationBus.models;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponseModel {

    private int statusCode;
    private String message;
    private String accessToken;
    private Long loginTime;
    private Long expirationTeime;



}
